www/js/app.js

```diff
@@ -393,6 +393,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             document.getElementById('modes').scrollIntoView();
         }, 600);
     });
+    document.getElementById('navigationButtons').ondragstart=function () {return false;}
```

Comment:
Please add a comment above to explain the purpose

Reply:
Done